layouts/api.html

```diff
@@ -10,7 +10,10 @@ <h2>Millennium</h2>
       <h2>Soarian</h2>
     <% end %>
     <ul>
-      <li><a href="/<%= @item[:solution] %>/dstu2/" class="active">DSTU 2 Final (1.0.2)</a></li>
+      <% if @item[:solution] == 'millennium' %>
+        <li><a href="/<%= @item[:solution] %>/overview/" class="active">Overview</a></li>
+      <% end %>
+      <li><a href="/<%= @item[:solution] %>/dstu2/">DSTU 2 Final (1.0.2)</a></li>
```

Comment:
do we need to keep the active class?

Reply:
Updated here: https://github.com/cerner/fhir.cerner.com/pull/396/commits/5fe2a063418c4b36151bec05d9b1279bb78b284e